pontoon/base/static/js/translate.js

```diff
@@ -498,11 +498,16 @@ var Pontoon = (function (my) {
      */
     generateLocalePluralExamples: function () {
       var self = this;
-      var examples = self.locale.examples = {0: 1, 1: 2};
       var nplurals = self.locale.nplurals;
-      var n = 0;
 
-      if (nplurals !== 2) {
+      if (nplurals === 2) {
+        self.locale.examples = {0: 1, 1: 2};
+      }
+
+      else {
+        var examples = self.locale.examples = {};
```

Comment:
Is it a common pattern in javascript to create 2 references and mutate only one of them? (I'm asking not judging).

Reply:
I just use `examples` as a shortcut. I hope that's ok.